src/install.rs

```diff
@@ -408,9 +419,48 @@ pub fn edit_bls_entries(mountpoint: &Path, f: impl Fn(&str) -> Result<String>) -
         }
     }
 
+    if found_path.is_none() {
+        bail!("Found no BLS entries in {}", config_path.display());
+    }
+
     Ok(())
 }
 
+/// Wrapper around `visit_bls_entry` to specifically visit just the BLS entry's `options` line and
+/// optionally update it if the function returns new content. Errors out if none or more than one
+/// `options` field was found.
```

Comment:
The latter seems to be a valid case according to specs, though.

Reply:
I think I'd prefer to error out. OSTree never emits multiple `options` and it actually doesn't even support parsing it (later `options` just override earlier ones).